bufr-config.cmake is expecting DA targets.

DA targets were removed, but the `cmake/PackageConfig.cmake.in` was not updated.
As a result, `find_package(bufr)` fails with the following error:
```
CMake Error at /opt/modules/gnu-10.3.0/bufr/da5eda3/lib/cmake/bufr/bufr-config.cmake:38 (get_target_property):
  get_target_property() called with non-existent target "bufr::bufr_4_DA".
Call Stack (most recent call first):
  CMakeLists.txt:45 (find_package)


CMake Error at /opt/modules/gnu-10.3.0/bufr/da5eda3/lib/cmake/bufr/bufr-config.cmake:44 (get_target_property):
  get_target_property() called with non-existent target "bufr::bufr_4_DA".
Call Stack (most recent call first):
  CMakeLists.txt:45 (find_package)
```

--- a/cmake/PackageConfig.cmake.in
+++ b/cmake/PackageConfig.cmake.in
@@ -1,8 +1,5 @@
 @PACKAGE_INIT@
 
-#  * @PROJECT_NAME@::@PROJECT_NAME@_4_DA - real32 library target with dynamic allocation
-#  * @PROJECT_NAME@::@PROJECT_NAME@_8_DA - real64 library target with dynamic allocation
-#  * @PROJECT_NAME@::@PROJECT_NAME@_d_DA - mixed library target with dynamic allocation
 #  * @PROJECT_NAME@::@PROJECT_NAME@_4    - real32 library target
 #  * @PROJECT_NAME@::@PROJECT_NAME@_8    - real64 library target
 #  * @PROJECT_NAME@::@PROJECT_NAME@_d    - mixed library target
@@ -11,11 +8,11 @@
 include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@-targets.cmake")
 
 # Get the build type from real32 library target with dyanmic allocation
-get_target_property(@PROJECT_NAME@_BUILD_TYPES @PROJECT_NAME@::@PROJECT_NAME@_4_DA IMPORTED_CONFIGURATIONS)
+get_target_property(@PROJECT_NAME@_BUILD_TYPES @PROJECT_NAME@::@PROJECT_NAME@_4 IMPORTED_CONFIGURATIONS)
 
 check_required_components("@PROJECT_NAME@")
 
 set(@PROJECT_NAME@_TABLES_DIR "@MASTER_TABLE_DIR@")
 
-get_target_property(location @PROJECT_NAME@::@PROJECT_NAME@_4_DA LOCATION)
+get_target_property(location @PROJECT_NAME@::@PROJECT_NAME@_4 LOCATION)
 message(STATUS "Found @PROJECT_NAME@: ${location} (found version \"@PROJECT_VERSION@\")")

